### PR TITLE
Alerting: Improve Amazon SNS documentation

### PR DESCRIPTION
**Pull request by JacobsonMT**
**What is this feature?**

Improves documentation around creating an Amazon SNS receiver in Grafana Alerting.

<details>
  <summary>Documentation Screenshot</summary>
  
Screen Shot 2024-09-26 at 18 33 08

</details>

### CODE REVIEW

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-amazon-sns.md [hunk 1]
```diff
@@ -55,3 +82,124 @@ To add this contact point to your alert, complete the following steps.
...
+{{< admonition type="note" >}}
+Replace the placeholder values (`https://sns.us-east-1.amazonaws.com`, `us-east-1`, `<YOUR_ACCESS_KEY>`, `<YOUR_SECRET_ACCESS_KEY>`, `arn:aws:iam::<account_id>:role/GrafanaSNSRole`, `arn:aws:sns:<region>:<account_id>:My-Topic`) with your actual AWS credentials and ARNs.
+{{< /admonition >}}
+
+## Additional Resources
+
+- [Amazon SNS Documentation](https://docs.aws.amazon.com/sns/index.html)
+- [AWS IAM Documentation](https://docs.aws.amazon.com/iam/index.html)
+- [Prometheus Alertmanager SNS Integration](https://prometheus.io/docs/alerting/configuration/#sns_config)
+- [Cloudwatch AWS Authentication]({{< relref "/docs/grafana/latest/datasources/aws-cloudwatch/aws-authentication" >}})
```
[the remark this answers is not in the document]
Comment (JacobsonMT) [reply docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-amazon-sns.md:205]: Hmm odd, the link appears to work with `make docs`.

**Comment by JacobsonMT** [comment]
Should we backport to 11.1.x and 11.2.x like the original? https://github.com/grafana/grafana/pull/92075

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-amazon-sns.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JacobsonMT) [reply docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-amazon-sns.md]: Moving to a relative link seems to fix it

**Comment by brendamuir** [comment]
Hey Matt - for linking because we use versioned docs for oss and also the docs in cloud - we follow this format:
Example -- 
In the front matter:
refs:
 time-units-and-relative-ranges:
   - pattern: /docs/grafana/
     destination: /docs/grafana/<GRAFANA_VERSION>/dashboards/use-dashboards/#time-units-and-relative-ranges
   - pattern: /docs/grafana-cloud/
     destination: /docs/grafana-cloud/visualizations/dashboards/use-dashboards/#time-units-and-relative-ranges

As a link in the text:
[time range](ref:time-units-and-relative-ranges)

**Review by brendamuir — APPROVED** [review]

**Pull request merged**